politeiad/cmd/politeia/politeia.go

```diff
@@ -58,12 +68,20 @@ func usage() {
 	fmt.Fprintf(os.Stderr, "  inventory         - Inventory records "+
 		"<vetted count> <branches count>\n")
 	fmt.Fprintf(os.Stderr, "  new               - Create new record "+
-		"<name> <filename>...\n")
+		"[mdid:metadata]... <filename>...\n")
 	fmt.Fprintf(os.Stderr, "  getunvetted       - Retrieve record "+
 		"<id>\n")
 	fmt.Fprintf(os.Stderr, "  setunvettedstatus - Set unvetted record "+
 		"status <publish|censor> <id>\n")
-	//fmt.Fprintf(os.Stderr, "  update      - Update record\n")
+	fmt.Fprintf(os.Stderr, "  update            - Update unvetted record "+
```

Comment:
The update parameters should include `token:<token>`.

Reply:
Yep good find.